src/app/asset-grid/asset-grid.component.ts

```diff
@@ -240,6 +240,12 @@ export class AssetGrid implements OnInit, OnDestroy {
     this.subscriptions.push(
       this.route.params.pipe(
       map((params: Params) => {
+
+        // if advanced search, inject search survey script
+        if (this.route.snapshot.params.advSearch) {
+          this.injectAdvancedSearchSurvey();
+        }
```

Comment:
Are we not able to import this into the search results component?

Reply:
what do you mean by "search results" component?

Reply:
Was thinking of this one https://github.com/ithaka/aiw-ui/blob/stage/src/app/search-page/search-page.component.ts

Reply:
🤔 yeah that's probably a more logical place
moved that conditional and method to `search-page.component.ts`